k8s/deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: revive-remix-backend
-        image: paritytech/revive-remix-backend:1.3.5
+        image: paritytech/revive-remix-backend:1.3.6
```

Comment:
What versioning is this following?

Reply:
Now Minor update is the change in backend logic
Revive change is the Patch update

Reply:
I think it would make sense to use the revive version in the tag?  Then you don't need to do this when actually nothing changed: There can be multiple version of the `revive-remix_backend` but the images include different `revive` versions. As a side effect it's directly visible what compiler version is deployed too.

Reply:
I think that it may be the last backend release, so probably it does not make sense to introduce any changes now.